python/lsst/obs/base/formatters/fitsGeneric.py

```diff
@@ -63,5 +63,8 @@ def add_provenance(
         else:
             # Unable to find compliant metadata attribute.
             return in_memory_dataset
-        add_provenance_to_fits_header(metadata, self.dataset_ref, provenance)
+        # Sometimes astropy is used to write FITS headers and astropy
+        # can not write long HIERARCH values. Therefore always elide some
```

Comment:
-> cannot?

Reply:
I guess. I don't like the word "cannot" and don't understand what's wrong with "can not".